server/apiRouter.js

```diff
@@ -80,4 +83,15 @@ router.get('/auth/google', authenticateGoogle);
 // loginWithIdp endpoint in Flex API to authenticate user to Flex
 router.get('/auth/google/callback', authenticateGoogleCallback);
 
+// This endpoint is called when user wants to initiate authenticaiton with Linkedin
+router.get('/auth/linkedin', authenticateLinkedin);
+
+// This is the route for callback URL the user is redirected after authenticating
+// with Linkedin. In this route a Passport.js custom callback is used for calling
+// loginWithIdp endpoint in Flex API to authenticate user to Flex
+router.get('/auth/linkedin/callback', authenticateLinkedinCallback);
+
+router.get('/.well-known/openid-configuration', openIdConfiguration);
```

Comment:
It would be good to have some comment or even link somewhere, where one can read more about these routes that this setup requires.